lib/options/SchemaDateOptions.js

```diff
@@ -48,6 +48,13 @@ Object.defineProperty(SchemaDateOptions.prototype, 'max', opts);
 /**
  * If set, Mongoose creates a TTL index on this path.
  *
+ * mongo TTL index `expireAfterSeconds` value will takes 'expires' value expressed in seconds.
```

Comment:
```suggestion
 * mongo TTL index `expireAfterSeconds` value will take 'expires' value expressed in seconds.
```
There's a typo `expires` => `expire`